motoman_driver/src/io_ctrl.cpp

```diff
@@ -65,7 +65,7 @@ bool MotomanIoCtrl::init(SmplMsgConnection* connection)
   return true;
 }
 
-bool MotomanIoCtrl::readSingleIO(shared_int address, shared_int &value)
+bool MotomanIoCtrl::readSingleIO(shared_int address, shared_int &value, std::string &err_msg)
```

Comment:
Not too happy with adding yet another *out* argument, but for now this works.

We'll probably want to refactor the IO client (to add support for IO groups and registers), so I did not want to change too much here in this PR.